llvm/test/Transforms/SampleProfile/overflow.ll

```diff
@@ -0,0 +1,72 @@
+; Checks that we are able to handle overflowing counters correctly.
+
+; RUN: opt < %s -passes=sample-profile -sample-profile-file=%S/Inputs/overflow.proftext | opt -passes='print<branch-prob>' -disable-output 2>&1 | FileCheck %s
+
+; Original Source:
+; int sqrt(int);
+; int lol(int i) {
```

Comment:
this function name doesn't match the IR... Though FWIW this seems so simple a function, can just as well remove the whole "Original Source" part in this case as far as I am concerned...